build/pxc-entrypoint.sh

```diff
@@ -197,6 +197,29 @@ if [[ $MYSQL_VERSION == '8.0' ]]; then
 	fi
 fi
 
+auth_plugin=${DEFAULT_AUTHENTICATION_PLUGIN}
+if [[ -f /var/lib/mysql/auth_plugin ]]; then
+	prev_auth_plugin=$(cat /var/lib/mysql/auth_plugin)
+	if [[ ${prev_auth_plugin} != "mysql_native_password" && ${auth_plugin} == "mysql_native_password" ]]; then
+		echo "FATAL: It's forbidden to switch from ${prev_auth_plugin} to ${auth_plugin}."
```

Comment:
I think we need to add a task number. The users should understand why it is forbidden.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1415/commits/9f2355a6aa2f7fd8cd9b6dbd9ab3521946633dad